app/components/Account/AccountOverview.jsx

```diff
@@ -252,6 +278,9 @@ class AccountOverview extends React.Component {
             const canDepositWithdraw = !!this.props.backedCoins.get("OPEN", []).find(a => a.symbol === asset.get("symbol"));
             const canWithdraw = canDepositWithdraw && (hasBalance && balanceObject.get("balance") != 0);
             const canBuy = !!this.props.bridgeCoins.get(symbol);
+            const changeClass = !marketsCache[asset.get("id")] ? "" : marketsCache[asset.get("id")][1] == 0 ? "" : marketsCache[asset.get("id")][1] < 0 ? marketsCache[asset.get("id")][2] ? "pulsate-down" : "change-down" : marketsCache[asset.get("id")][2] ? "pulsate-up" : "change-up";
```

Comment:
I don't think this line does what you think it does. The syntax is incorrect (too many : parts like here: `? "pulsate-down" : "change-down" : marketsCache[asset.get("id")][2]`) and marketsCache[asset.get("id")][1] is a string so you should parse it before comparing it to 0.